### DISCUSSION TITLE
Welcome to CPlusPlus-PatternHouse Discussions!

### DISCUSSION THREAD

**Discussion by aryashah2k** [PatternHouse/CPlusPlus-PatternHouse#680] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aryashah2k** [comment]
@anshyyy you can go ahead and ask your queries here buddy!

**Comment by anshyyy** [comment]
hello @aryashah2k ,
I really want to know how to solve the issues(numeric pattern) because there's no description. can you please provide some description about issue? Numeric Pattern 328

**Reply by aryashah2k** [reply]
Sure buddy, so PatternHouse has a lot of Patterns that require to be coded, in Numeric Patterns there are 328 patterns, so the issue will be resolved when you code the pattern

Now the question is how do you know whatvthe pattern is?
You can check the readme file in individual folder and then :
1. See the pattern
2. Go to issues tab
3. search the relevant issue
4. mention me there that you want to contribute
5. I will assign that issue to you
6. You can then start working on the code and create PR
7. IMPORTANT: the naming convention needs to be followed for every code file so that the frontend of our website is able to register that 
Naming convention is : "numericpattern1.cpp"

**Comment by aryashah2k** [comment, marked as answer]
You can also go to our website: https://portal.patternhouse.tk/numeric.html

Find the pattern number which you want to contribute and then come in the repository issues tab and search the issue from search box and tag me to get it assigned to you!
